parachain/pallets/ethereum2-light-client/src/mock.rs

```diff
@@ -0,0 +1,50 @@
+use frame_support::{
+	parameter_types,
+	traits::{Everything, GenesisBuild},
+};
+use frame_system as system;
+
+use crate as verifier;
+
+pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+
+pub mod mock_verifier {
+
+	use super::*;
+
+	type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+	type Block = frame_system::mocking::MockBlock<Test>;
+
+	frame_support::construct_runtime!(
+		pub enum Test where
+			Block = Block,
+			NodeBlock = Block,
+			UncheckedExtrinsic = UncheckedExtrinsic,
+		{
+			System: frame_system::{Pallet, Call, Storage, Event<T>},
+			Verifier: verifier::{Pallet, Call, Config, Storage, Event<T>},
```

Comment:
Obsolete naming. In our production runtimes we refer to the module as `EthereumLightClient` instead of `Verifier`

```suggestion
			EthereumLightClient: crate::{Pallet, Call, Config, Storage, Event<T>},
```